src/introduction/cross.md

```diff
@@ -0,0 +1,10 @@
+# Cross compilation and execution
+
+For the moment, for the majority of users, the primary way of obtaining or
+creating CHERI programs is *cross-compilation*.  This is always something of a
+chore, but we have attempted to make it as painless as possible for the
```

Comment:
Maybe we should clarify why it’s a chore; perhaps: “This is always something of a chore, as most software has been implemented to run on the same architecture on which it has been built. However, we have attempted...” -- or something more elegantly written.